index.js

```diff
@@ -177,16 +207,20 @@ var compile = function(schema, cache, root, reporter, opts) {
             JSON.stringify(schemaPath)
           )
         } else {
-          validate('validate.errors.push({field:%s,message:%s})', formatName(prop || name), JSON.stringify(msg))
+          validate(
+            'validate.errors.push({field:%s,message:%s})',
+            formatName(prop || name),
+            JSON.stringify(msg)
+          )
         }
       }
     }
 
     if (node.default !== undefined) {
       indent++
-      validate('if (%s === undefined) {', name)
-                ('%s = %s', name, jaystring(node.default))
-              ('} else {')
+      validate('if (%s === undefined) {', name)('%s = %s', name, jaystring(node.default))(
+        '} else {'
+      )
```

Comment:
I dislike this formatting change because it makes it harder to read. OK for this PR, will fix in future